sum-of-nth-terms.py

```diff
@@ -0,0 +1,15 @@
+def series_sum(n):
+    if n == 0:
+        return "%.2f" % n
+    elif n == 1:
+        return "%.2f" % n
+    else:
+        denom = 1
+        num1 = 0
+        num2 = 0
+        for i in range(0, n - 1):
+            denom = denom + 3
+            num1 = 1/denom
+            total = 1 + num1 + num2
```

Comment:
You're doing your arithmetic using integers. In Python 2, integer division always returns an integer, and rounds down. So, in Python 2, this function will **always** return "1.00".